### ISSUE TITLE
pkg_resources is deprecated

### ISSUE THREAD

**Issue by tandav** [no-plagiarism/pymorphy3#19]
**Describe the bug**
Running pytest in the project which uses pymorphy3 gives warning:

```
  /home/ubuntu/.cache/virtualenvs/myenv/lib/python3.10/site-packages/pymorphy3/analyzer.py:114: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
    import pkg_resources
```

**Comment by insolor** [comment]
There is a fix for this warning in other fork: https://github.com/pymorphy2-fork/pymorphy2/pull/19.

But it is for Python 3.8 and later, so probably older versions of Python should be dropped first (or the solution should be expanded for more python versions).

**Comment by BLKSerene** [comment]
This issue could be closed as already solved in 2.0.1.

**Comment by tandav** [comment]
Thank you, closing the issue
